boards/riscv/beagleV/doc/index.rst

```diff
@@ -0,0 +1,71 @@
+.. _beagelV:
+
+BeagleV
+#######
+
+Overview
+********
+
+The BeagleV is an 64-bit open-source RISC-V development board with
+a StarFive JH7100 SoC.
+
+Programming and debugging
+*************************
+
+Building
+========
+
+Applications for the ``beagleV`` board configuration can be built as usual
+(see :ref:`build_an_application`) using the corresponding board name:
+
+.. zephyr-app-commands::
+   :board: beagleV
```

Comment:
In general, I don't see many boards (if any) using camel case. I think most of them are all lower case.

Perhaps someone else can chime in to see if lowercase board names are a policy or convention or just coincidence.

Reply:
Yes beagle_v sounds like a nice compromise if we can't have camel case names. 
One more thing, I guess it should be bealge_v_evb and for the production board, it should be simple beagle_v. 
it would be great if someone can confirm the camel case thing.

Reply:
Changed the name to beagle_v_starlight.